presto-docs/src/main/sphinx/release/release-0.269.rst

```diff
@@ -0,0 +1,37 @@
+=============
+Release 0.269
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add :func:`uuid` to return a unique identifier.
+* Add JDBC URL parameter doc for MySQL.
+
+Presto on Spark Changes
+_______________________
+* Add a new configuration property ``spark.memory-revoking-target`` to specify the percent to which memory pool should be occupied after revoke is completed.  This can be overridden by ``spark_memory_revoking_target`` session property.
+
+Accumulo Connector Changes
+__________________________
+* Replace :func:`uuid` from being connector specific to a standard function. In the process the return type has been changed from ``VARCHAR`` to ``UUID`` and existing use cases might need to perform a cast - ``CAST(UUID() AS VARCHAR)``.
+
+Delta Table Connector Changes
+_____________________________
+* New connector for reading Delta Lake tables(https://delta.io/) natively in Presto.
+
+Iceberg Connector Changes
+_________________________
+* Add Iceberg table predicate pushdown to reduce the number of files scanned.
+
+Pinot Connector Changes
+_______________________
+* Adding Pinot session property ``pinot.topn_large`` to override the default topN value.
```

Comment:
Adding -> Add
also, maybe this makes more sense to someone more familiar with pinot, but what is "the default topN value" (both, what's the number, but also what does it mean/ how is it used, topN what?)

Reply:
@highker Since you merged the code, could you provide more details here so I can add to the release notes?

Reply:
we can remove this one